src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -507,6 +507,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('exclude_path', exclude_path_type)
         c.argument('include_path', include_path_type)
         c.argument('recursive', recursive_type)
+        c.argument('content_type', arg_group='Additional Flags', help="Specify content type of the file. "
+                   "Implies no-guess-mime-type.")
```

Comment:
Maybe we can use imperative mood for alignment? Also, "Implies" isn't very intuitive to me. Possibly "If unspecified, ..."?

Reply:
sure. updated.

Reply:
We misunderstood it! 😰 It means if `--content-type` is given, `--no-guess-mime-type` is implied.